translation.rst

```diff
@@ -351,9 +351,7 @@ Translation Resource/File Names and Locations
 Symfony looks for message files (i.e. translations) in the following default locations:
 
 * the ``translations/`` directory;
-
-* the ``src/Resources/<bundle name>/translations/`` directory;
-
+* the ``translations/bundles/<bundle name>/`` directory;
```

Comment:
Are you sure about this?

Reply:
https://github.com/symfony/symfony/blob/4f9dc9d44988448441da48269793e7cd3fa4b949/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L898-L900

However, the path was wrong (without `bundles/`), thanks! Fixed!